plugins/GSdx/GSTextureCache.cpp

```diff
@@ -480,6 +480,10 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 			GSVector2i custom_resolution = m_renderer->GetCustomResolution();
 			scale_factor.x = static_cast<float>(custom_resolution.x) / width;
 			scale_factor.y = static_cast<float>(custom_resolution.y) / height;
+
+			//Don't bother setting a scale factor when people request same resolution as native resolution.
+			if(custom_resolution == GSVector2i(width, height))
+				scale_factor = GSVector2(0,0);
```

Comment:
> Wrong again :P , what would happen if the scale factor is set to 0 ?
> 
> I have a rough idea on what you'll reply next ;)

Then it's still scaling down from 1 isn't it :P  

you could put a check in set scale to make sure it's not 0 if you wanted to, but that shouldn't be a possible value to enter :P